examples/gradle/errorprone/experimental_errors

```diff
@@ -0,0 +1,27 @@
+errorProneExperimentalErrors = \
```

Comment:
Can we share this with the errorprone files from the root project?

Reply:
Yes. I'll remove these files.